app/containers/Applications/ApplicationDetailsCardItem.tsx

```diff
@@ -43,11 +46,21 @@ export const ApplicationDetailsCardItemComponent: React.FunctionComponent<Props>
     >
       <Card.Header onClick={() => setIsOpen(!isOpen)}>
         <Row>
-          <Col md={9}>
-            <h2>{formJsonByFormId.name}</h2>
+          <Col md={6}>
+            <h4>{formJsonByFormId.name}</h4>
           </Col>
-          <Col md={3} style={{textAlign: 'right'}}>
-            <Button>{isOpen ? 'Expand' : 'Collapse'}</Button>
+          {isAnalyst ? (
```

Comment:
Instead on conditionally rendering the `ApplicationReviewContainer`, the page component should pass a `rendertemHeaderContent` function to the `ApplicationDetails` component, something like that: 
```jsx
(formResult) => <ApplicationReviewContainer formResult={formResult} />
```